features/image-transformations/image-enhancement-and-color-manipulation.md

```diff
@@ -133,3 +133,117 @@ URL - [https://ik.imagekit.io/demo/lipstick_2M.png](https://ik.imagekit.io/demo/
+|-----------------|-------------------|-------------------|------------|-------------------|
+| Linear Direction (`ld`) | **Optional**.<br/><br/>This is used to specify the direction vector of the linear gradient.<br/><br/>For enhanced control over the direction, this parameter also accepts a integer `0-359`, which specifies the angle (in degrees) when measured over a standard x-y axes with pivot passing through the geometrical centre of the image. For example, `0` would mean `top`, `45` would mean `top_right`, `90` would mean `right` etc. | `Number` or `String` | **Number** - `0` to `359`<br/><br/>**String** - `top`, `top_right`, `right`, `bottom_right`, `bottom`, `bottom_left`, `left`, `topleft` | `180` (or `bottom`) |
+| From Color (`from`) | **Optional**.<br/><br/>This is used to specify a *from* color for the gradient.<br/><br/>RGBA Code (e.g. `FFAABB50`) is a color representation model where the first six characters must be the RGB Hex Code and the last two characters must be a number between `00` and `99`, which indicates the opacity level. For example, `00` represents an opacity level of `0.00`, `01` represents an opacity level of `0.01`, and so on. | `String` | An [SVG recognized color](https://www.w3.org/TR/SVG11/types.html#ColorKeywords) keyword name eg. `aqua`;<br/><br/>or a RGB Hex Code eg. `FF0000`;<br/><br/>or a RGBA Hex Code eg. `FF000040`; | `FFFFFF` |
+| To Color (`to`) | **Optional**.<br/><br/>This is used to specify a *to* color for the gradient.<br/><br/>RGBA Code (e.g. `FFAABB50`) is a color representation model where the first six characters must be the RGB Hex Code and the last two characters must be a number between `00` and `99`, which indicates the opacity level. For example, `00` represents an opacity level of `0.00`, `01` represents an opacity level of `0.01`, and so on. | `String` | An [SVG recognized color](https://www.w3.org/TR/SVG11/types.html#ColorKeywords) keyword name eg. `aqua`;<br/><br/>or a RGB Hex Code eg. `FF0000`;<br/><br/>or a RGBA Hex Code eg. `FF000040`; | `00000000` |
+| Stop Point (`sp`) | **Optional**.<br/><br/>This is used to dictate where the break point should be made between the `from` color and the `to` color. In simpler words, it is the point (along the direction vector of the gradient) at which the `to` color achieves it’s pure form i.e. without any blends.<br/><br/>When the number is less than or equal to 1 i.e. a floating point, it would be treated as a percentage value. For example, `0.25` means 25%, `0.1` means 10% and so on.<br/><br/>When the number is greater than 1, it would be treated as an absolute value (in pixels). For example, `100` would mean 100px and so on. | `Number` or `String` | **Number** - A floating point number between `0.01` and `0.99` or an integer greater than or equal to `1`.<br/><br/>**String** - An arithmetic expression like `ih_div_2`. Read more about arithmetic expressions [here](../arithmetic-expressions-in-transformations.md). | `1` |
```

Comment:
```suggestion
| Stop Point (`sp`) | **Optional**.<br/><br/>This is used to dictate where the break point should be made between the `from` color and the `to` color. In simpler words, it is the point (along the direction vector of the gradient) at which the `to` color achieves it’s pure form i.e. no blending with the `from` color. <br/><br/>When the number is less than or equal to 1 i.e. a floating point, it would be treated as a percentage value. For example, `0.25` means 25%, `0.1` means 10% and so on.<br/><br/>When the number is greater than 1, it would be treated as an absolute value (in pixels). For example, `100` would mean 100px and so on. | `Number` or `String` | **Number** - A floating point number between `0.01` and `0.99` or an integer greater than or equal to `1`.<br/><br/>**String** - An arithmetic expression like `ih_div_2`. Read more about arithmetic expressions [here](../arithmetic-expressions-in-transformations.md). | `1` |
```